### ISSUE TITLE
Execution blocked jxbrowser chromium.exe

### ISSUE THREAD

**Issue by dyardy** [flutter/flutter-intellij#6358]
I am using JetBrains IntelliJ and the flutter plugin.

Our company blocks the usage of browsers and, in this particular case, is blocking the execution of 
C:\Users\{username}\AppData\Local\JxBrowser\7.27\chromium.exe

I am currently using flutter plug 70.2.5
and IntelliJ 2022.2.3

I am using Intellij to build/run Flutter applications.  

Can I disable the plugin from running this chromium.exe? (do I need this?)
Can I change it to use Google Chrome from the default installation directory?

[√] Flutter (Channel stable, 3.3.4, on Microsoft Windows [Version 10.0.19044.1889], locale en-US)
    • Flutter version 3.3.4 on channel stable at c:\src\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision eb6d86ee27 (3 weeks ago), 2022-10-04 22:31:45 -0700
    • Engine revision c08d7d5efc
    • Dart version 2.18.2
    • DevTools version 2.15.0

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at c:\android\sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at:
      C:\Users\dyardy\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\213.7172.25.2113.9123335\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Enterprise 2022 17.3.6)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Enterprise
    • Visual Studio Enterprise 2022 version 17.3.32929.385
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2021.3)
    • Android Studio at C:\Users\dyardy\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\213.7172.25.2113.9123335
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)

[√] IntelliJ IDEA Ultimate Edition (version 2022.2)
    • IntelliJ at C:\Users\dyardy\AppData\Local\JetBrains\Toolbox\apps\IDEA-U\ch-0\222.4345.14
    • Flutter plugin version 70.2.5
    • Dart plugin version 222.4345.14

[√] VS Code, 64-bit edition (version 1.72.2)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension can be installed from:
       https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (4 available)
    • sdk gphone64 x86 64 (mobile) • emulator-5554 • android-x64    • Android 13 (API 33) (emulator)
    • Windows (desktop)            • windows       • windows-x64    • Microsoft Windows [Version 10.0.19044.1889]
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 106.0.5249.119
    • Edge (web)                   • edge          • web-javascript • Microsoft Edge 106.0.1370.52

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

**Comment by stevemessick** [comment]
You can use the built-in inspector by unchecking the check box at the bottom of the Flutter plugin settings page.

To run the embedded inspector, I believe it has to use chromium, not chrome. @helin24 might have a better answer.

**Comment by stevemessick** [comment]
Does using the built-in inspector work?

**Comment by dyardy** [comment]
Within Settings > Languages & Frameworks > Flutter > Experiments
I unchecked 'Enable embedding DevTools in the Flutter Inspector tool window' and now when I run/debug it does not prompt me about chromium being blocked/prevented.  This is an improvement.

Yes, the tool window 'Flutter Inspector' does work.  Keep in mind I am using physical device via usb.

Earlier I tried emulators on a new company laptop and performance was really slow.  Not sure if I can improve or not.  The physical device seems much smoother.
